origin/reg_status.go

```diff
@@ -60,7 +60,7 @@ type (
 // The TTL depends on the Status: For Status == StatusRegistrationError, the cache item won't expire,
 // meaning a failure registration stays in the cache, until the registration is successful.
 // For other Status, TTL is set to 5min
-var registrationsStatus = ttlcache.New(ttlcache.WithTTL[string, RegistrationStatus](5 * time.Minute))
+var registrationsStatus = ttlcache.New(ttlcache.WithTTL[string, RegistrationStatus](1 * time.Minute))
```

Comment:
Does switching this to a 1-minute TTL negate the above comment?

Reply:
Thanks for the catch! Fixed